samcli/lib/providers/sam_api_provider.py

```diff
@@ -285,3 +308,17 @@ def merge_routes(collector):
             len(result),
         )
         return list(result)
+
+    @staticmethod
+    def _sort_key_stack_depth_desc(route: Route):
```

Comment:
use the `reverse` flag of `sorted` instead of having explicit descending order

Reply:
changed as in previous comment